reports/generator/generator.go

```diff
@@ -245,14 +241,8 @@ func (g *generator) getSimpleValueFromData(data []*map[string]interface{}, fullE
 			case int:
 				s = strconv.Itoa(val.(int))
 			case float32:
-				if float64(val.(float32)) == math.Trunc(float64(val.(float32))) {
```

Comment:
This has been removed because variable `s` set in if statement is always overwritten by 
```s = strconv.FormatFloat(float64(val.(float32)), 'G', -1, 64)```
 on [line 251/244](https://github.com/iryonetwork/wwm/blob/e42cf12fff7de3d59a5dd186070300967e2afff8/reports/generator/generator.go#L251)

Reply:
Ok, right, my bad :) Please just add else then and fix my mistake.